src/CONST.ts

```diff
@@ -2113,6 +2113,15 @@ const CONST = {
             AUTOREPORTING_OFFSET: 'autoReportingOffset',
             GENERAL_SETTINGS: 'generalSettings',
         },
+        // Upgradeable modules that don't need outside connections
+        INTERNAL: {
+            NAME: {
+                APPROVALS: 'approvals',
+            },
+            NAME_USER_FRIENDLY: {
+                approvals: 'Advanced Approvals',
+            },
+        },
```

Comment:
I think it's better to remove this whole object:
```suggestion
```

And instead use regular properties likfe for `glCodes`, `glAndPayrollCodes` in `UPGRADE_FEATURE_INTRO_MAPPING`, would that work?

Reply:
yes, I think so, just changed it!